submissions/kotlyar-andrey/dom/style.css

```diff
@@ -0,0 +1,372 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+html,
+body {
+  height: 100%;
+  background-color: #212121;
+}
+@font-face {
+  font-family: "Cormorant Unicase";
```

Comment:
Always add fallback font. [Read more](https://css-tricks.com/css-basics-fallback-font-stacks-robust-web-typography/)

Reply:
Not fixed :cry: